incubator/oom-event-generator/values.yaml

```diff
@@ -0,0 +1,54 @@
+# Default values for oom-event-generator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# repliaCount -- The number of replicas to run
+replicaCount: 1
+
+image:
+  repository: xingse/kubernetes-oom-event-generator
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v1.2.0"
```

Comment:
The standard helm chart that I think you used automatically grabs the `appVersion` from the chart.yaml, so I think you can leave this empty.

Reply:
appVersion is `1.2.0`
tag is `v1.2.0` -- there is no tag `1.2.0` per docker hub

Reply:
Why not set appVersion to the image tag then? That way we don't have to maintain both.